source/table-filters/apply-table-filters.txt

```diff
@@ -26,28 +26,42 @@ clause, a row limit clause, or both.
 Steps
 -----
 
+.. procedure::
```

Comment:
This is largely a cleanup effort, to add the procedure directive for symmetry with the other pages, there are no content edits.